docs/deployment-examples/nginx-on-nix-deployments/configure-octopus-deploy-project.md

```diff
@@ -0,0 +1,161 @@
+
+![](/docs/deployment-examples/nginx-on-nix-deployments/images/deployment_process_name_role_and_package.png "width=500")
+
+### NGINX Web server
+
+To configure NGINX to send traffic to your application you need to fill in a few details.
+
+| Field                     | Meaning                                  | Examples                                 | Notes                                    |
+| ------------------------- | ---------------------------------------- | ---------------------------------------- | ---------------------------------------- |
+| **Host Name**             | The `Host` header that this server will listen on. | `www.contoso.com`              | **[Optional]** The value can be a full (exact) name, a wildcard, or a regular expression. A wildcard is a character string that includes the asterisk (`*`) at its beginning, end, or both; the asterisk matches any sequence of characters. Leave empty to use any `Host` header. |
+| **Bindings**              | Specify any number of HTTP/HTTPS bindings that should be added to the NGINX virtual server. |                                          |                                          |
+| **Locations**             | Specify any number of locations that NGINX should test request URIs against to send traffic to your application. |                                          |                                          |
+
+When defining **locations** you can configure NGINX to deliver files from the file system , or proxy requests to another server. For our sample application we want requests to `http://<IPorDNSofServer>/` to deliver the `index.html` file from the `wwwroot` folder of our ASP.NET Core project and requests to `http://<IPorDNSofServer>/api/` to be proxied to our ASP.NET Core project running on http://localhost:5000.
+
+![](/docs/deployment-examples/nginx-on-nix-deployments/images/deployment_process_nginx_feature.png "width=500")
+
+### Starting and Managing Our ASP.NET Core project
+
+To get the ASP.NET Core process started up you can manually call *dotnet <nameofaspnetcoreapplication>.dll*, however this has its drawbacks when trying to run the process in the background of your deployment environments. Each time you deploy a new version of the package you would then have to stop the old version and start the newly deployed one. Without running the process through some intermediary process manager you would need to search for and kill the previous one from the process list, based on something like parsing its path to determine the correct one. This is obviously fraught with dangers. A better approach is to use a process manager, for the purposes of this simple example we will use `systemd` (as nearly all Linux distributions use this process manager) to demonstrate how the web process might be managed.
```

Comment:
> This is obviously fraught with dangers.

I don't think this sentence is needed. Things are not always obvious if you haven't done them before.  The preceding sentence hints at the difficulty of manually managing processes and the next sentence presents a better alternative so I think you can leave this sentence out without losing anything.